### PR TITLE
[SIG-3178] add info text to CategorySelect

### PR DESCRIPTION
**Pull request by aditudorache**
This PR adds the info text to the CategorySelect object.

### CODE REVIEW

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
 import Select from 'components/SelectInput';
 import { useSelector } from 'react-redux';
 import { makeSelectSubCategories } from 'models/categories/selectors';
+import InfoText from 'components/InfoText';
+
+const Wrapper = styled.div`
+  width: 100%;
```
Comment (janjaap) [inline src/signals/incident/components/form/CategorySelect/CategorySelect.js:11]: A div, by default, has a width of `100%`. Why is it necessary to set it here explicitly?

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+  margin-bottom: 0;
+  p {
```
Comment (janjaap) [inline src/signals/incident/components/form/CategorySelect/CategorySelect.js:13]: How do you know that a `p` tag is rendered as a child of `Wrapper`? Looking at the code of the `CategorySelect` component, that isn't obvious.

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js [hunk 3]
```diff
@@ -1,19 +1,42 @@
...
+  const { value } = handler();
+
+  const [info, setInfo] = useState();
+
+  const getSubcategory = useCallback(slug => subcategories?.find(
+    s => s.slug === slug
+  ) || {}, [subcategories]);
+
+  useEffect(() => {
+    const { description } = getSubcategory(value);
```
Comment (janjaap) [inline src/signals/incident/components/form/CategorySelect/CategorySelect.js:32]: What if `getSubcategory` returns `undefined`?

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident/components/form/CategorySelect/CategorySelect.js:11]: It isn't :-), I removed it.

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js (hunk 3 above)
Comment (aditudorache) [reply src/signals/incident/components/form/CategorySelect/CategorySelect.js:32]: It shouldn't. But I added extra tests.

File: src/signals/incident/components/form/CategorySelect/CategorySelect.js (hunk 2 above)
Comment (aditudorache) [reply src/signals/incident/components/form/CategorySelect/CategorySelect.js:13]: I moved the styling to the InfoText component

**Review by janjaap — APPROVED** [review]

**Pull request merged**